.github/workflows/etc/requirements.txt

```diff
@@ -18,5 +18,6 @@ h5py==3.11.0
 
 # testing
 pytest==8.3.2
+pytest-qt==4.4.0
```

Comment:
Suppose it was only a matter of time before this amazing pytest plugin made it's way into the library; with it being added, we can certainly use more of its capabilities (in another PR)

Reply:
👍 

Note that I ensured it is an _optional_ dependency at this point: all tests can be run without it.

Reply:
Appreciate that, but it won't be long before it's mandatory. Would love to remove some of the stuff in our ui testing files and offload to pytest-qt!